### DISCUSSION TITLE
7680502A-397D-45C8-ACC9-A9F27C1FAC1C

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#176] [General]
# 7680502A-397D-45C8-ACC9-A9F27C1FAC1C

Chandrayaan-2 and 3 are ISRO robotic missions to the Moon in 2019 and 2023 respectively. Upon orbit insertion, both missions each sent a Vikram lander and a Pragyan rover to the lunar south pole. Only the latter flight is successful, making India the fourth country to successfully land on the Moon. Both the Chandrayaan orbiters are still active to this day, while CH-3's lander has already completed its 14-day surface mission and is now decommissioned

https://celestia.mobi/addon?item=7680502A-397D-45C8-ACC9-A9F27C1FAC1C

**Comment by OlivaxNoel** [comment]
Good job India!
